app/src/main/java/covid/trace/morocco/PreferencesExtensions.kt

```diff
@@ -0,0 +1,28 @@
+package covid.trace.morocco
+
+import android.content.SharedPreferences
+import androidx.core.content.edit
+
+inline fun <reified T> SharedPreferences.get(key: String?, defaultValue: T): T {
```

Comment:
I like this syntax.
`key` can be non optional according to Android docs.

Reply:
Resolved